osu.Game/Graphics/OsuFont.cs

```diff
@@ -20,6 +20,41 @@ public static class OsuFont
         /// </summary>
         public static FontUsage Default => GetFont();
 
+        /// <summary>
+        /// Equivalent to Torus with 32px size and semi-bold weight.
+        /// </summary>
+        public static FontUsage Display => GetFont(size: 32, weight: FontWeight.SemiBold);
```

Comment:
What does "Display" mean here? I think we need to choose a better term.

Reply:
It seems to be a terminology in fonts indicating large text, but it's more commonly used for font variation rather than sizes. Alternative names can be `Headline` or `Title`.

Reply:
Title and subtitle work fine enough. You don't need to invent new standards.

![Safari 2025-04-07 at 10 12 22](https://github.com/user-attachments/assets/8b82e595-2742-4619-9d2d-6c651e775be3)

![Safari 2025-04-07 at 10 14 03](https://github.com/user-attachments/assets/bf81d030-b3f5-4b29-9908-18163c3d2d1f)